### DISCUSSION TITLE
Fixed length encoded string

### DISCUSSION THREAD

**Discussion by d3bt3ch** [sqids/sqids-java#11] [Q&A] [answered]
Instead of minimum length is it possible to have a fixed length encoded string?

**Comment by 0x3333** [comment, marked as answer]
Unfortunately, it can't be done. This is a limitation of the Sqids itself, not something tied to this implementation.

**Reply by 4kimov** [reply]
More info: https://sqids.org/faq#maxlength

**Reply by d3bt3ch** [reply]
Got it 👍
